samples/javascript_typescript/60.multilingual-luis-bot/translator.ts

```diff
@@ -0,0 +1,345 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import * as request from 'request-promise-native';
+import { promises } from 'fs';
+
+
+export class TranslateArrayOptions{
```

Comment:
Use interfaces or types for this use case. We represent data classes in TS mostly by interfaces or types